app/main/views/g_cloud.py

```diff
@@ -276,6 +276,25 @@ def search_services():
     )
 
 
+@direct_award.route('', methods=['GET'])
+def saved_search_overview():
+    projects = data_api_client.find_direct_award_projects(current_user.id).get('projects', [])
+    open_searches = []
+    closed_searches = []
+
+    for project in projects:
+        if project['active']:
+            open_searches.append(project)
```

Comment:
Let's call these `open_projects` and `closed_projects` to keep terminology consistent.